datasource/src/test/scala/quasar/physical/s3/S3DataSourceSpec.scala

```diff
@@ -106,6 +118,20 @@ class S3DataSourceSpec extends DatasourceSpec[IO, Stream[IO, ?]] {
         data_12_34)
     }
 
+    "read array JSON of resource with special chars in path" >>* {
+      assertEvaluate(
+        datasource,
+        ResourcePath.root() / ResourceName("testData") / ResourceName("á") / ResourceName("βç.json"),
```

Comment:
How do we get `βç.json` into the bucket? Curious since `βç.json` is part of this PR.

Reply:
I uploaded it manually. I assumed the other, already existing test files were also manually uploaded?

Reply:
yes, they were. We probably want to fix that eventually.